src/apy/config.py

```diff
@@ -34,21 +35,19 @@ def get_base_path():
     cfg = {}
 
 
+CFG_DEFAULT_VALUES: dict[str, Any] = {
+    "base_path": None,
+    "img_viewers": {
+        "svg": ["display", "-density", "300"],
+    },
+    "img_viewers_default": ["feh"],
+    "markdown_models": ["Basic"],
+    "presets": {},
+    "profile_name": None,
+    "query": "tag:marked OR -flag:0",
+}
```

Comment:
Was this change necessary, or only useful to indicate the point of the dict? Also, I think I'd prefer an empty line after line 48 here.

Reply:
mypy was complaining because it needed an explicit annotation for `default` in the line below:

```
for required, default in ...
```

I guess I could have just solved it by adding one line:

```
default: Any
```

but I feel using a dictionary instead of a tuple conveys better the idea of it being a default configuration.

Wdyt?

(I will add a new line)

Reply:
I think this looks good; thanks!